test/e2e/etcdctl_test.go

```diff
@@ -0,0 +1,116 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TestEtcdctlCommands executes all known etcdctl commands inside of the etcdctl container.
+// The test is not intended to be a functional test yet a sanity test that the container
+// ENV is populated correctly and that etcdctl consumes that ENV properly.
+func TestEtcdctlCommands(t *testing.T) {
```

Comment:
Although I personally hate Ginkgo, I expected this to be part of an exiting Ginkgo suite using the usual test framework for consistency and integration with the overall e2e test scaffolding

Reply:
yeah I mean if this were origin I would 100% agree its just not a standard pattern across any of the control-plane operators. In fact the usage across the org is almost nil. the regular testing frameworks is light and saves a pretty large dependency. I am open to considering it.

https://github.com/search?q=org%3Aopenshift+ginko&type=code

Reply:
Well if I spelled it right would help... so the usage is a bit more than I expected.

https://github.com/search?q=org%3Aopenshift+ginkgo&type=code

Reply:
Apologies... I think when I reviewed this before I was totally spaced out and confused — I thought I was looking at a test in the origin repo and didn't realize we were talking about etcd-operator e2e tests. Nevermind everything I said re: Ginkgo, sorry for the confusion.

Reply:
no worries I know you like to standardize stuff :+1: maybe it should be? I think if operator e2e were to evolve maybe it would need that framework. Let's keep it in mind.